paywall/src/services/storageService.js

```diff
@@ -36,6 +43,7 @@ export default class StorageService {
         hash: t.transactionHash,
         network: t.chain,
         to: t.recipient,
+        input: t.data,
```

Comment:
How would you feel about adjusting `input` to `data` for consistency?

Reply:
we can't do that, because web3Service expects the field to be named input